test/python/test_load_qasm.py

```diff
@@ -25,20 +26,20 @@ class LoadQasmTest(QiskitTestCase):
     """Test load_qasm_* set of methods."""
 
     def setUp(self):
-        self.QASM_FILE_NAME = 'entangled_registers.qasm'
-        self.QASM_FILE_PATH = self._get_resource_path(
-            'qasm/' + self.QASM_FILE_NAME, Path.EXAMPLES)
+        self._qasm_file_name = 'entangled_registers.qasm'
```

Comment:
I went for the public pattern almost in all other places. I did this because I was testing something which is that the refactor utilities take a lot less to refactor private variables (since they don't need to check beyond the class scope). I'm fine with making it public but it is worth noticing.